### ISSUE TITLE
C and P arguments are not explained

### ISSUE THREAD

**Issue by come-maiz** [etiennepinchon/aframe-glow#3]
Please explain in the README what do C and P mean.

Thanks a lot for sharing this module!

**Comment by etiennepinchon** [comment]
Hey! Yes you are right I will add a little note, basically they are the 2 parameters from the shader responsible for the glow. You can test those two values here: http://stemkoski.github.io/Three.js/Shader-Glow.html
Hope this helps! :)

**Comment by come-maiz** [comment]
Yes, thanks @etiennepinchon. I have been playing with the values, but I don't have it clear enough to document it myself :)

Sadlly, what I was trying to make glow is a transparent png image, and that doesn't look very well:

https://github.com/JaquerEspeis/khipu-360/issues/6#issuecomment-328701567

But for 3d geometries, it works very nicely.

**Comment by etiennepinchon** [comment]
Sure no problem! 
Yeah it is not perfect just yet, it does not work on 3D models as well. Hopefully this will be resolve soon!

**Comment by come-maiz** [comment]
For the record, I found out that I can put my png inside a sphere, and get the effect I was looking for. <3

![ekrankopio de 2017-09-13 13-01-10](https://user-images.githubusercontent.com/617831/30395555-cd86a312-9883-11e7-8435-2e75ea9b9dbd.png)

**Comment by etiennepinchon** [comment]
Oh nice dude! Thanks for sharing!
